base_layer/wallet_ffi/tests/api/test_wallet_api.py

```
@@ -0,0 +1,415 @@
+            for network in invalid_networks:
+                try:
+                    invalid_params = {
+                        'config': base_config,
+                        'log_path': os.path.join(temp_dir, f"invalid_{network}"),
+                        'log_verbosity': 1,
+                        'num_rolling_log_files': 2,
+                        'size_per_log_file_bytes': 256*1024,
+                        'network_str': network
+                    }
+                    
+                    wallet = tari_wallet.PyTariWallet(**invalid_params)
```

Ruff (0.11.9)

Local variable `wallet` is assigned to but never used

Remove assignment to unused variable `wallet`

(F841)